third_party/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+if(POLICY CMP0091)
+  cmake_policy(SET CMP0091 NEW)
+endif()
+
+project(thirdparty CXX)
+set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreadedDLL")
```

Comment:
Please remove the DLL suffix.

Reply:
`MultiThreadedDLL` -> `/MD` and `MultiThreaded` -> `/MT`. We use `/MT` here.

Reply:
Yup this was my mistake. Should be fixed now